main.go

```diff
@@ -186,26 +186,26 @@ func mainE(config Configs, appPatterns []string) error {
 		return fmt.Errorf("failed to find artifacts, error: %v", err)
 	}
 
-	var artNames []string
+	var artPaths []string
 	for _, a := range artifacts {
-		artNames = append(artNames, a.Name)
+		artPaths = append(artPaths, a.Name)
 	}
 
-	log.Donef("Found artifacts by provided pattern:")
+	log.Donef("Used patterns for generated ")
```

Comment:
i feel like smtg is missing from here `"Used patterns for generated `

Reply:
Fixed in:
f3e3002